### ISSUE TITLE
500 error on saving page in Elementor editor

### ISSUE THREAD

**Issue by blakmarkit** [WPDevelopers/essential-addons-for-elementor-lite#737]
WP pages made with the classic/gutenberg editor save without error. Upon saving from within the Elementor editor, a 500 server error toast pops up. Something about the new asset loading performance improvements introduced in 5.2 seems to be conflicting with WP's post revisions. This is happening on an environment running on LocalWP as well as on WP Engine.

```
PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function Essential_Addons_Elementor\\Classes\\Asset_Builder::delete_cache_data(), 1 passed in /nas/content/live/<site>/wp-includes/class-wp-hook.php on line 287 and exactly 2 expected in /nas/content/live/<site>/wp-content/plugins/essential-addons-for-elementor-lite/includes/Classes/Asset_Builder.php:428
Stack trace:
#0 /nas/content/live/<site>/wp-includes/class-wp-hook.php(287): Essential_Addons_Elementor\\Classes\\Asset_Builder->delete_cache_data(3267)
#1 /nas/content/live/<site>/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters('', Array)
#2 /nas/content/live/<site>/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#3 /nas/content/live/<site>/wp-includes/post.php(3070): do_action('after_delete_po...', 3267)
#4 /nas/content/live/<site>/wp-includes/revision.php(441): wp_delete_post(3267)
#5 /nas/content/live/<site>/wp-includes/revision.php(217): wp_delete_post_revision(3267)
#6 /nas/content/live/<site>/wp-includes/class-wp-hook.php in /nas/content/live/<site>/wp-content/plugins/essential-addons-for-elementor-lite/includes/Classes/Asset_Builder.php on line 428, referer: https://<site>.com/wp-admin/post.php?post=3265&action=elementor
```

WP Core: 5.4.10
Host: WP Engine
Server Memory Limit: 512MB
Theme: Hello Elementor + Custom child theme
Elementor: 3.7.3
Elementor Pro: 3.7.4
Essential Addons Lite: 5.2.4
Essential Addons Pro: 5.2.0

**_Edit to add, downgrading EA Lite to 5.1.9 appears to resolve the issue._**

**Comment by blakmarkit** [comment]
This issue also appears to be resolved by upgrading Essential Addons Pro to 5.2.1, for both EA Lite 5.1.9 as well as the latest, 5.2.4. 

Closing as no longer relevant.

**Comment by blakmarkit** [comment]
I unfortunately stand corrected. Sorry for the whiplash. The error started appearing again. Reopening the ticket.

**Comment by rudlinkon** [comment]
Thank you for your info, our dev team is working on it. We will let you know soon.

**Comment by abid112** [comment]
Hi @blakmarkit , we've fixed this issue on our plugin (in **v5.3.0**). I'm closing this issue as resolved!
